modules/search-service-impl/src/main/java/org/opencastproject/search/endpoint/SearchRestService.java

```diff
@@ -111,6 +114,10 @@ public class SearchRestService extends AbstractJobProducerEndpoint {
 
   private final Gson gson = new Gson();
 
+  private UrlSigningService urlSigningService;
+
+  private long expireSeconds = UrlSigningServiceOsgiUtil.DEFAULT_URL_SIGNING_EXPIRE_DURATION;
```

Comment:
Setting this to `DEFAULT_URL_SIGNING_EXPIRE_DURATION` implies that this is configurable. Otherwise, you could also have used the constant in `findURLsAndSign(…)` directly. But this is never updated. Is this supposed to be configurable?

Reply:
I'm merging it like this to get it into 16.3. It works and shouldn't hurt after all. I've filed this issue as https://github.com/opencast/opencast/issues/6099. If you can, it would be nice if you would take a look at that issue.

Reply:
Ok cool! You can assign the issue to me if you like ( i cannot do that myself)